medmodels/treatment_effect_estimation/tests/test_treatment_effect.py

```diff
@@ -0,0 +1,369 @@
+"""Tests for the TreatmentEffect class in the treatment_effect module."""
+
+import unittest
+
+import pandas as pd
+
+from medmodels import MedRecord
+from medmodels.medrecord import node
+from medmodels.treatment_effect_estimation.treatment_effect import TreatmentEffect
+
+
+def create_patients() -> pd.DataFrame:
+    """
```

Comment:
FYI. I would say it is not necessary to include docstrings in functions that are not publicly accessible via the package and where it is already clear what the function is doing

Reply:
No need to change this however. Only for future reference